### DISCUSSION TITLE
[FTL] Error, no templates were found

### DISCUSSION THREAD

**Discussion by MichaelShoemaker** [projectdiscovery/nuclei#613] [Q&A] [answered]
**Describe the bug**
The templates listed below return the error "Error, no templates were found"
fuzzing
headless
helpers
miscellaneous
network
vulerabilities
workflows

When I rename the directories they work, but when I rename them back to their original names the error appears again. All other template directories run without error. 

**Nuclei version**
nuclei - Current Version: 2.2.0
OS - Kali 2020.4

**Screenshot of the error or bug**
Here you see it cannot find the template
![NucleiTemplateNotWorking](https://user-images.githubusercontent.com/7443591/111040164-659ed180-83f7-11eb-95bb-42f0bb637678.png)

After renaming the directory it runs fine
![NucleiRenameWorking](https://user-images.githubusercontent.com/7443591/111040208-74858400-83f7-11eb-835c-b4a2e972b92b.png)

Renaming the directory back to the original name causes the error to appear again
![NucleiNotWorkingAgain](https://user-images.githubusercontent.com/7443591/111040212-794a3800-83f7-11eb-8017-01db5805bf50.png)

**Comment by ehsandeep** [comment, marked as answer]
Hey @MichaelShoemaker, 

1. make sure to update the nuclei, v2.3.0 is the latest, [release blog](https://blog.projectdiscovery.io/nuclei-v2-3-0-release/.)
2. the behavior you are describing here is a feature, [details here](https://nuclei.projectdiscovery.io/nuclei/get-started/#template-exclusion)
3. you can just edit the `~/.nuclei-ignore` file to configure what you want to run as default, entries in this will exclude templates from default.

**Reply by MichaelShoemaker** [reply]
@ehsandeep Sorry for my confusion and thank you for the quick response. I'm a noob with nuclei and a bit slow. Update to version 2.3.0 by rerunning:
`GO111MODULE=on go get -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei`
and edited the excluded templates in the file
`nuclei-templates/.nuclei-ignore`
Just being explicit because it took me a few minutes to figure out so if someone else comes across this it might help them out. Thank you again!

**Reply by ehsandeep** [reply]
np @MichaelShoemaker, Feel free to use the Github discussion / Discord server linked in this project in case you got more questions or doubts.

**Comment by anggipradana** [comment]
in windows .nuclei-ignore always resets to default when nuclei is run even though it was changed before, is there a solution for this?
